src/wfdlgs2.c

```diff
@@ -188,59 +188,62 @@ SearchDlgProc(register HWND hDlg, UINT wMsg, WPARAM wParam, LPARAM lParam)
+                      BOOL bMaximized = FALSE;
+                      HWND hwndMDIChild = (HWND)SendMessage(hwndMDIClient, WM_MDIGETACTIVE, 0, (LPARAM)&bMaximized);
+
+                      // cf. https://www.codeproject.com/articles/2077/creating-a-new-mdi-child-maximization-and-focus-is
+                      if (bMaximized)
+                      {
+                          SendMessage(hwndMDIClient, WM_SETREDRAW, FALSE, 0);
+                      }
 
-					  // cf. https://www.codeproject.com/articles/2077/creating-a-new-mdi-child-maximization-and-focus-is
-					  if (bMaximized)
-					  {
-						  SendMessage(hwndMDIClient, WM_SETREDRAW, FALSE, 0);
-					  }
+                      //
+                      // !! BUGBUG !!
+                      //
+                      // This is safe since szMessage = MAXPATHLEN*2+MAXSUGGESTLEN
+                      // but it's not portable
+                      //
+                      LoadString(hAppInstance, IDS_SEARCHTITLE, szMessage,
+                          COUNTOF(szMessage));
 
-                     //
-                     // !! BUGBUG !!
-                     //
-                     // This is safe since szMessage = MAXPATHLEN*2+MAXSUGGESTLEN
-                     // but it's not portable
-                     //
-                     LoadString(hAppInstance, IDS_SEARCHTITLE, szMessage,
-                        COUNTOF(szMessage));
+                      lstrcat(szMessage, SearchInfo.szSearch);
 
-                     lstrcat(szMessage, SearchInfo.szSearch);
+                      // Have the MDIClient create the MDI directory window.
 
-                     // Have the MDIClient create the MDI directory window.
+                      MDICS.szClass = szSearchClass;
+                      MDICS.hOwner = hAppInstance;
+                      MDICS.szTitle = szMessage;
 
-                     MDICS.szClass = szSearchClass;
-                     MDICS.hOwner = hAppInstance;
-                     MDICS.szTitle = szMessage;
+                      // Minimize
 
-                     // Minimize
+                      MDICS.style = bMaximized ? WS_MAXIMIZE : WS_MINIMIZE;
+                      MDICS.x = CW_USEDEFAULT;
+                      MDICS.y = 0;
+                      MDICS.cx = CW_USEDEFAULT;
+                      MDICS.cy = 0;
 
-                     MDICS.style = bMaximized ? WS_MAXIMIZE : WS_MINIMIZE;
-					 MDICS.x  = CW_USEDEFAULT;
-                     MDICS.y  = 0;
-                     MDICS.cx = CW_USEDEFAULT;
-                     MDICS.cy = 0;
+                      SendMessage(hwndMDIClient, WM_MDICREATE, 0L, (LPARAM)(LPMDICREATESTRUCT)&MDICS);
 
-                     SendMessage(hwndMDIClient, WM_MDICREATE, 0L, (LPARAM)(LPMDICREATESTRUCT)&MDICS);
+                      // Forward the attributes to search window
+                      SetWindowLongPtr(hwndSearch, GWL_ATTRIBS, GetWindowLongPtr(hwndMDIChild, GWL_ATTRIBS));

```

Comment:
Make a note that hwndSearch is created by the above call with WM_MDICREATE.

Reply:
done